src/database/migrations/20191022190917-add_comment.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.createTable('Comments',{
+      id: {allowNull: false,autoIncrement: true, primaryKey: true, type: Sequelize.INTEGER },
+      comment: {type:Sequelize.STRING, allowNull:false},
+      userId: {
+        type: 'integer',
+        allowNull: true,
+        defaultValue: 0,
```

Comment:
Great job. However I don't think this should allow null, the default value is unnecessary in this case. there should be a user associated with any comment